graylog2-web-interface/src/views/stores/useViewsSelector.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import type { RootState } from 'views/types';
+import useAppSelector from 'stores/useAppSelector';
+
+const useViewsSelector = <R>(fn: (state: RootState) => R) => useAppSelector<R, RootState>(fn);
+export default useViewsSelector;
```

Comment:
I had to create a new hook for the selector and dispatcher, so we can make use of the related views types and don't have to repeat them. 

I named these hooks and the store provide specific to views, because we are in the views module, but it might be confusing, because the store contains a view slice.